grafana/dashboards/statistics.json

```diff
@@ -695,6 +751,35 @@
               "type": "macro"
             }
           ]
+        },
+        {
+          "datasource": {
+            "type": "grafana-postgresql-datasource",
+            "uid": "TeslaMate"
+          },
+          "editorMode": "code",
+          "format": "table",
+          "hide": false,
+          "rawQuery": true,
+          "rawSql": "WITH data AS (\r\n    SELECT\r\n        p.car_id,\r\n        date_trunc('$period', TIMEZONE('UTC', p.date)) as date,\r\n        lag(p.${preferred_range}_battery_range_km) OVER (ORDER BY p.date) - p.${preferred_range}_battery_range_km AS range_loss,\r\n        p.odometer - lag(p.odometer) OVER (ORDER BY p.date) AS distance\r\n    from positions p\r\n    WHERE\r\n      p.car_id = $car_id and\r\n      p.ideal_battery_range_km is not null and\r\n      $__timeFilter(p.date)\r\n    ORDER BY date\r\n)\r\n\r\nSELECT\r\n  EXTRACT(EPOCH FROM date)*1000 AS date_from,\r\n  EXTRACT(EPOCH FROM date + interval '1 $period')*1000 AS date_to,\r\n  CASE '$period'\r\n    WHEN 'month' THEN to_char(date, 'YYYY Month')\r\n    WHEN 'year' THEN to_char(date, 'YYYY')\r\n    WHEN 'week' THEN 'week ' || to_char(date, 'WW') || ' starting ' || to_char(date, 'YYYY-MM-DD')\r\n    ELSE to_char(date, 'YYYY-MM-DD')\r\n  END AS display,\r\n  date,\r\n  SUM(range_loss * car.efficiency * 1000) / convert_km(sum(distance)::numeric, '$length_unit') as consumption_gross_$length_unit\r\n  FROM data\r\n  JOIN cars car on car.id = car_id\r\n  WHERE distance >= 0 and range_loss >= 0\r\n  GROUP BY date",
+          "refId": "D",
+          "sql": {
+            "columns": [
+              {
+                "parameters": [],
+                "type": "function"
+              }
+            ],
+            "groupBy": [
+              {
+                "property": {
+                  "type": "string"
+                },
+                "type": "groupBy"
+              }
+            ],
+            "limit": 50
+          }
```

Comment:
this has been added to correctly calculate `Ø Consumption (gross)` within a period. Numbers should look more in line with other dashboards now.